mappings/net/minecraft/recipe/Ingredient.mapping

```diff
@@ -5,30 +5,32 @@ CLASS net/minecraft/class_1856 net/minecraft/recipe/Ingredient
 	FIELD field_9019 entries [Lnet/minecraft/class_1856$class_1859;
 	METHOD <init> (Ljava/util/stream/Stream;)V
 		ARG 1 entries
-	METHOD method_8086 fromPacket (Lnet/minecraft/class_2540;)Lnet/minecraft/class_1856;
+	METHOD method_26964 create (Ljava/util/stream/Stream;)Lnet/minecraft/class_1856;
+		ARG 0 stream
+	METHOD method_8086 read (Lnet/minecraft/class_2540;)Lnet/minecraft/class_1856;
```

Comment:
We cannot take a refactor is it changes one name fitting the guideline to another one that does not comply to the guideline. :-1: